Platform/ElkhartlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -581,6 +581,10 @@ UpdateFspConfig (
     Fspmcfg->PchHdaIDispLinkTmode       = GfxCfgData->PchHdaIDispLinkTmode;
     Fspmcfg->PchHdaIDispCodecDisconnect = GfxCfgData->PchHdaIDispCodecDisconnect;
 
+    if ((FeaturePcdGet (PcdPreOsCheckerEnabled) && PchIsSciSupported ()) && !FixedPcdGetBool (PcdIsReleaseBuild)) {

```

Comment:
What about using PlatformFeatures->DebugConsent to enable/disable audio?

Reply:
DebugConsent cant justify for debug build of Fusa sku. There might be a case for non Fusa sku with DebugConsent value of 7.